### ISSUE TITLE
Cyrillic glyphs

### ISSUE THREAD

**Issue by rellikmil** [mjlagattuta/Hepta-Slab#12]
Wow, marvelous slab serif. But without Cyrillic

**Comment by Bronnel721** [comment]
I agree. Because there are Cyrillic currency symbols but no Cyrillic.
EDIT: Not to long ago I realized that in the medium master of the source file, there is Cyrillic but it's incomplete.
